src/components/views/voip/VideoView.js

```diff
@@ -50,7 +50,10 @@ module.exports = React.createClass({
     },
 
     getRemoteAudioElement: function() {
-        return this.refs.remoteAudio;
+        // this needs to be somewhere at the top of the DOM which
+        // always exists to avoid audio interruptions.
+        // Might as well just use DOM.
+        return document.getElementById("remoteAudio");
```

Comment:
Oh, I was looking on `master` which is why I didn't see it.

I would prefer passing it down (it's more work but it makes it more reusable). If you don't want to do that, I'm equally happy with you handling the `null` case and whining very loudly in the console rather than just blindly NPEing if the element doesn't exist, along with a comment explaining where this ID is expected to come from.